files/default/sql/ddl/3.2.0__initial_tables.sql

```diff
@@ -2356,3 +2356,57 @@ CREATE TABLE IF NOT EXISTS `hdfs_command_execution` (
   CONSTRAINT `fk_hdfs_file_command_2` FOREIGN KEY (`src_inode_partition_id`,`src_inode_pid`,`src_inode_name`) REFERENCES `hops`.`hdfs_inodes` (`partition_id`, `parent_id`, `name`) ON DELETE CASCADE
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `feature_monitoring_configuration` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `feature_group_id` INT(11),
+    `feature_view_id` INT(11),
+    `feature_name` VARCHAR(63) COLLATE latin1_general_cs NOT NULL,
+    `enabled` BOOLEAN DEFAULT TRUE,
+    `feature_monitoring_type` tinyint(4) NOT NULL,
+    `alert_config` VARCHAR(63) COLLATE latin1_general_cs NOT NULL, -- dummy this should become ref to another table
+    `scheduler_config` VARCHAR(63) COLLATE latin1_general_cs NOT NULL, -- dummy this should become ref to another table
+    PRIMARY KEY (`id`),
+    KEY (`feature_name`),
+    CONSTRAINT `fg_monitoring_config_fk` FOREIGN KEY (`feature_group_id`) REFERENCES `feature_group` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION,
+    CONSTRAINT `fv_monitoring_config_fk` FOREIGN KEY (`feature_view_id`) REFERENCES `feature_view` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE = ndbcluster DEFAULT CHARSET = latin1 COLLATE = latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `monitoring_window_builder` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `feature_monitoring_config_id` INT(11) NOT NULL,
+    `window_builder_type` INT(11) NOT NULL,
+    `detection` BOOLEAN NOT NULL,
+    `specific_id` INT(11),
+    `time_offset` VARCHAR(63),
+    `window_length` VARCHAR(63),
+    `row_percentage` INT(11),
+    `specific_value` FLOAT,
+    PRIMARY KEY (`id`),
+    CONSTRAINT `fm_window_builder_fk` FOREIGN KEY (`feature_monitoring_config_id`) REFERENCES `feature_monitoring_configuration` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE = ndbcluster DEFAULT CHARSET = latin1 COLLATE = latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `descriptive_statistics_monitoring` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `feature_monitoring_config_id` INT(11) NOT NULL,
+    `strict` BOOLEAN DEFAULT FALSE,
+    `relative` BOOLEAN DEFAULT FALSE,
+    `threshold` FLOAT,
+    `compare_on` INT(11) NOT NULL,
+    PRIMARY KEY (`id`),
+    CONSTRAINT `fm_descriptive_stats_fk` FOREIGN KEY (`feature_monitoring_config_id`) REFERENCES `feature_monitoring_configuration` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE = ndbcluster DEFAULT CHARSET = latin1 COLLATE = latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `feature_monitoring_result` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `feature_monitoring_config_id` INT(11) NOT NULL,
+    `job_id` INT(11) NOT NULL, -- dummy this should become ref to another table
+    `execution_id` INT(11) NOT NULL,  -- dummy this should become ref to another table
+    `monitoring_time` timestamp DEFAULT CURRENT_TIMESTAMP,
+    `triggered_alert` BOOLEAN DEFAULT FALSE,
```

Comment:
Just a reminder here, about other naming proposals for this field. Maybe `shifted`, `shift_detected`, `out_of_bounds`?@DhananjayMukhedkar

Reply:
Yes I would agree, `triggered_alert` could be confusing as a name

Reply:
Ja choose one together that you think is the right one I will put it in accordingly